jrnl/util.py

```diff
@@ -108,7 +107,12 @@ def set_keychain(journal_name, password):
         except keyring.errors.PasswordDeleteError:
             pass
     else:
-        keyring.set_password("jrnl", journal_name, password)
+        try:
+            keyring.set_password("jrnl", journal_name, password)
+        except keyring.errors.NoKeyringError:
+            print(
```

Comment:
Please print errors to `stderr` ([example](https://github.com/jrnl-org/jrnl/blob/develop/jrnl/util.py#L53)).